src/app/shared/services/login/login.service.ts

```diff
@@ -138,30 +143,34 @@ export class LoginService {
       );
   }
 
-  private handleLoginErrors(e) {
+  /**
+   *  Handles non descriptive errors from iam-client-lib and cache server.
+   *  TODO: think about better implementation/approach.
+   */
+  private handleLoginErrors(e, navigateOnTimeout) {
     console.error(e);
     const loginError = LOGIN_ERRORS.filter(error => e.message.includes(error.key))[0];
     if (loginError?.type === 'swal') {
-      this.displayLoginErrorWithSwal(loginError.value);
+      const config = {
+        title: 'Wrong Network',
+        text: `${loginError.value}`,
+      };
```

Comment:
It's not obvious to me why `loginError?.type === 'swal'` mentions that the error is `Wrong Network`. Does it make sense to you?

Reply:
Yeah, i added todo to rethink this error messaging for login. I will try today or tomorrow prepare some better approach to this.

Reply:
More fundamentally then the message I think, I don't really understand how `loginError?.type === 'swal'` communicates anything about the actual error. Isn't `swal` just "SweetAlert", which is just how the message is displayed? So how can we understand the substance of the error if the check is for the error display approach 😕 ? Maybe I'm missing something here though 😅

Reply:
Yes, swal is for sweetalert package. Second one is `toastr`. My first idea was to split it into 2 ways of handling this: one with sweetalert, second with toastr. The message should be taken from `loginError.value` and which type of message from `loginError.type`. It is not the best idea imo right now. I will rewrite this :)

Reply:
Ok, do you want do that as a part of this PR, or is it important to get this PR merged in? I think it might be important to get this merged in before deploying Dev Switchboard to Volta as this login issue is quite common

Reply:
What do you think?

Reply:
I think i can update it in this PR. I do not know when we are going to deploy Switchboard to Volta, but this shouldn't take more than few hours of coding. Anyway in this sprint I'm mostly testing + fixing bugs that i find in SB.

Reply:
Ok, let me know when to review the PR again then